plugins/workflow/notification/forms/workflow_notification.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="utf-8"?>
+<form>
+	<fields name="options">
+		<fieldset name="notification">	
+			<field
+				name="send_mail"
```

Comment:
The names should probably have a notification prefix? ```notification_send_mail```

Reply:
Until now, field names in the core have no prefix
and one case with prefix: actionlogsNotify (no underscore)

If we want a prefix, then at least in workflow it should be used everywhere in the same format.

Reply:
Still it should be ```notification_send_mail``` to avoid conflicts with other potential plugins. Same for other options.

Reply:
done